app/views/layouts/application.html.erb

```diff
@@ -120,38 +120,38 @@
           </div>
         </div>
       </div>
-    </footer>
 
-    <div id="global-footer" role="contentinfo">
```

Comment:
Multiple `role="contentinfo"` elements bad. Move the SU footer into the HTML footer.